README.md

```diff
@@ -6,22 +6,18 @@ Conceptual and machine-readable versions of the MEx metadata model.
 
 ## project
 
-With the Metadata Exchange (MEx) project, the [RKI](https.//www.rki.de) is developing a
-transparency platform for finding metadata on the institute's research activities and
-data.
+The Metadata Exchange (MEx) project is committed to improve the retrieval of RKI research data and projects. How? By focusing on metadata: instead of providing the actual research data directly, the MEx metadata catalog captures descriptive information about research data and activities. On this basis, we want to make the data FAIR* so that it can be shared with others.
 
-Via MEx, metadata will be made findable, accessible and shareable on a daily basis,
-as well as available for further research. The goal is to get an overview of what
-research data is available, understand its context, and know what needs to be considered
-for subsequent use.
+Via MEx, metadata will be made findable, accessible and shareable, as well as available for further research. The goal is to get an overview of what research data is available, understand its context, and know what needs to be considered for subsequent use.
 
-The platform is currently in internal trials but will also be made publicly available
-and thus be available to external researchers as well as the interested (professional)
-public.
+After an internal launch, the metadata will also be made publicly available and thus be available to external researchers as well as the interested (professional) public to find research data from the RKI.
```

Comment:
perhaps: "After an internal launch, parts of the metadata will also be made..." (the public will probably get a different "view" on the metadata than internal stakeholders?